test/utils.sh

```diff
@@ -557,3 +557,109 @@ replace_image_pullspec() {
     exit 2
   fi
 }
+
+# This function will be used by tasks in tekton-integration-catalog
+# Given the output of 'opm render $fbc_fragment' command, this function returns a package name
+# If there is only one 'olm.package', it returns it's name
+# If there are multiple 'olm.package' entries, it returns the one with the highest bundle version in the package’s 'defaultChannel'
+get_package_from_catalog() {
+  local RENDER_OUT_FBC="$1"
+  local package_count
+  local package_name
+
+  if [ -z "$RENDER_OUT_FBC" ]; then
+    echo "get_package_from_catalog: Missing 'opm render' output for the image" >&2
+    exit 2
+  fi
+
+  # Count 'olm.package' entries in the rendered FBC output
+  package_count=$(echo "$RENDER_OUT_FBC" | tr -d '\000-\031' | jq -s '[.[] | select(.schema == "olm.package")] | length')
+
+  if [[ "$package_count" -eq 1 ]]; then
```

Comment:
curious, what if there's zero 'olm.package' in the output? or is it even possible?

Reply:
Thank you for the clarification!